pkg/providers/instancetype/suite_test.go

```diff
@@ -663,8 +670,8 @@ var _ = Describe("InstanceType Provider", func() {
 			pod.Spec.Affinity = &v1.Affinity{NodeAffinity: &v1.NodeAffinity{PreferredDuringSchedulingIgnoredDuringExecution: []v1.PreferredSchedulingTerm{
 				{
 					Weight: 1, Preference: v1.NodeSelectorTerm{MatchExpressions: []v1.NodeSelectorRequirement{
-						{Key: v1.LabelTopologyZone, Operator: v1.NodeSelectorOpIn, Values: []string{fakeZone1}},
-					}},
+					{Key: v1.LabelTopologyZone, Operator: v1.NodeSelectorOpIn, Values: []string{fakeZone1}},
+				}},
 				},
```

Comment:
Weird indent?

Reply:
I added some newlines to this format to make it a bit more obvious what it was doing and fixed this wonky indentation mistake.